### ISSUE TITLE
Version for e-book readers

### ISSUE THREAD

**Issue by macie** [SquareBracketAssociates/UpdatedPharoByExample#209]
PDF format on e-book readers is _unpleasant_ (it cannot scale, you read every page like a image: zoom in - scroll - zoom out - next page). So it will be great to have a book in e-book reader freindly version, for example in [EPUB](https://en.wikipedia.org/wiki/EPUB).

Even the one-page HTML could be fine (with help of external apps like: **[calibre](https://calibre-ebook.com/)**).

**Comment by cdlm** [comment]
This is a general [Pillar issue](https://github.com/pillar-markup/pillar/issues/8). In fact there should already be some code for EPUB support, and help on maintaining it would be very welcome.

**Comment by macie** [comment]
Thanks for answer.
As I understand, the Pillar EPUB exporter is in quite early stage, and it will be easier to create one-page HTML. Am I right?

**Comment by cdlm** [comment]
Yes. In that case, the way to go would be to upgrade the book to the more recent build system used in the booklets, and then do `make htmlbook`. We're still cleaning the architecture of Pillar however, so that will again change soonish.

A quick and dirty alternative would be to simply merge the per-chapter HTML files together. If you just want to read the book now, that might be the simplest one…
